.vscode/settings.json

```diff
@@ -11,16 +11,15 @@
 
 	// Silent the stylistic rules in you IDE, but still auto fix them
```

Comment:
SilenCE the stylistic rules in youR IDE, but still auto fix them.
Just spelling

Reply:
![17398135785493622744898014290758](https://github.com/user-attachments/assets/ee9052c8-7003-4f31-ab13-b4d273d427c8)